locales/en.json

```diff
@@ -29,13 +31,13 @@
 	"However, since you follow someone that follows them, you should be able to see their latest posts.": "However, since you follow someone that follows them, you should be able to see their latest posts.",
 	"Mutual Friends": "Mutual Friends",
 	"You share %s mutual friends with this person.": {
-	  "one": "You share %s mutual friend with this person.",
-	  "other": "You share %s mutual friends with this person."
+		"one": "You share %s mutual friends with this person.",
+		"other": "You share %s mutual friends with this person."
 	},
 	"Followed by": "Followed by",
 	"You follow %s people that follow this person.": {
-	  "one": "You follow someone that follows this person.",
-	  "other": "You follow %s people that follow this person."
+		"one": "You follow %s people that follow this person.",
+		"other": "You follow %s people that follow this person."
```

Comment:
I'm not sure why these changes were made. It is overriding the custom strings with default ones. I will attempt to roll these back before merging. Hopefully my git foo is strong enough!!